src/main/java/uk/gov/di/ipv/domain/ErrorResponse.java

```diff
@@ -8,7 +8,9 @@
     ERROR_1000(1000, "Missing query parameters for auth request"),
```

Comment:
I'd suggest renaming these enum values so that they're easy to choose from where they're used, eg.

```
MissingQueryParameters(1000, "Missing query parameters for auth request"),
MissingRedirectURI(1001, "Redirect URI is missing from auth request"),
```
etc.

Reply:
Yeah that's a good shout. I'll change all the enum values